### PR TITLE
Instance cleanup e2e

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

E2E tests for the instance deregistration feature.

### CODE REVIEW

File: .photofinish.toml [hunk 1]
```diff
@@ -291,4 +288,40 @@ files = [
     "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4_host_discovery.json",
     "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4_sap_system_discovery.json",
     "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4_subscription_discovery.json",    
+]
+
+[NWD-00-present]
```
Comment (rtorrero) [inline .photofinish.toml:293]: I've noticed we usually start these with `sapsystem-` or `database-` or `host-`... depending on what we are pushing. In this case it's obvious but I'd add it to keep the good habit I guess :-)

**Review by EMaksy — COMMENTED** [review]
Hey code wise LGTM :)

File: test/e2e/cypress/e2e/databases_overview.cy.js [hunk 2]
```diff
@@ -48,4 +48,67 @@ context('Databases Overview', () => {
...
+      },
+    };
+
+    before(() => {
+      cy.contains(hddDatabase.sid).should('exist');
+
+      cy.get('table.table-fixed > tbody > tr').eq(0).click();
+    });
+
+    it('should mark an instance as absent and restore it as present', () => {
```
Comment (rtorrero) [inline test/e2e/cypress/e2e/databases_overview.cy.js:67]: I'm missing a "when" or an "after" to describe the condition

**Review by rtorrero — APPROVED** [review]
LGTM, 2 minor things commented

**Pull request merged**